<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientMainWindow</class>
 <widget class="QMainWindow" name="ClientMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>739</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Chat client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Настройки подключения</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Пользователь</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edtUserName">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>221</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Адрес сервера</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Порт</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edtAddres">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>221</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="edtPort">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>120</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnConnect">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>120</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="adtPassword">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>221</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Пароль</string>
     </property>
    </widget>
   </widget>
   <widget class="QListView" name="lstContacts">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>171</y>
      <width>381</width>
      <height>361</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>540</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddContact">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить контакт</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDelContact">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>540</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить контакт</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>edtUserName</tabstop>
  <tabstop>adtPassword</tabstop>
  <tabstop>edtAddres</tabstop>
  <tabstop>edtPort</tabstop>
  <tabstop>btnConnect</tabstop>
  <tabstop>btnAddContact</tabstop>
  <tabstop>btnDelContact</tabstop>
  <tabstop>btnExit</tabstop>
  <tabstop>lstContacts</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
